src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -476,6 +477,10 @@ bool layout_optimizer::can_fuse_reorder_to_prev(program_node& prev, reorder_node
         if (fmt_prev.dimension() > 6 || fmt_next.dimension() > 6)
             return false;
 
+        // Skip reorder fusing to permute when allow_new_shape_infer is True
+        if (allow_new_shape_infer)
+            return false;
```

Comment:
Sorry but please add one more condition. I'd like to add "input and output rank is different" to this. Could you please apply it? thanks!

Reply:
Updated! Thanks!